src/components/Carousel/props.js

```diff
@@ -48,6 +48,7 @@ export const propTypes = {
   hasDotButtonsAtMax: largeWidgetPositions,
   hasCaptionsAtMax: largeWidgetPositions,
   hasThumbnailsAtMax: PropTypes.bool,
+  placeholderImg: PropTypes.string,
```

Comment:
I'd suggest renaming it to `fallbackImg`.

Reply:
And also in other props.

Reply:
Done.